mod_ci/controllers.py

```diff
@@ -1005,27 +993,33 @@ def start_ci():
             if workflow_name in [Workflow_builds.LINUX, Workflow_builds.WINDOWS]:
                 g.log.debug('workflow_run event detected')
                 commit_hash = payload['workflow_run']['head_sha']
-                github_status = repository.statuses(commit_hash)
+                github_status = repository.get_commit(commit_hash)
 
                 if payload['action'] == "completed":
                     is_complete = True
                     has_failed = False
                     builds = {"linux": False, "windows": False}
-                    for workflow in repository.actions.runs.get(
+
+                    # NOTE: Using this workaround because of bug in PyGithub
```

Comment:
Is this an open bug? Better refer to it then, so we can easily check in the future if it's fixed or not.

Reply:
If not reported yet, have you reported it?

Reply:
I checked the bug is not reported yet; I'll create an issue there and reference it here
Also, I am yet to write unit tests for this PR.

Reply:
@canihavesomecoffee @thealphadollar 
The issue already existed: https://github.com/PyGithub/PyGithub/issues/2276; I have also referenced this in the code.
The unit tests for this PR are also complete now.